pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -726,6 +730,29 @@ def default_compilation_pass(self, optimisation_level: int = 2) -> BasePass:
                     RemoveRedundancies(),
                 ]
             )
+        else:
+            passlist.extend(
+                [
+                    RemoveBarriers(),
```

Comment:
Should we also issue a warning if the circuit contains barriers?

Reply:
good point, added to the new overloaded `get_compiled_circuit` method